app/src/main/java/com/keylesspalace/tusky/components/notifications/NotificationsFragment.kt

```diff
@@ -217,22 +218,14 @@ class NotificationsFragment :
             }
         })
 
-        /**
-         * Collect this flow to notify the adapter that the timestamps of the visible items have
-         * changed
-         */
+        // update post timestamps
         val updateTimestampFlow = flow {
-            while (true) { delay(60000); emit(Unit) }
-        }.onEach {
-            layoutManager.findFirstVisibleItemPosition().let { first ->
-                first == RecyclerView.NO_POSITION && return@let
-                val count = layoutManager.findLastVisibleItemPosition() - first
```

Comment:
Refresh all elements otherwise with scrolling you will easily see list elements with a wrong timestamp.